metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -194,12 +194,21 @@ spec:
     rules:
     - alert: ODFRBDClientBlocked
       annotations:
-        description: A kernel RBD client on node {{ $labels.node_name }} is blocklisted by Ceph for more than 10s. Please check Ceph logs.
-        message: A kernel RBD client on node {{ $labels.node_name }} is blocklisted by Ceph.
+        description: An RBD client might be blocklisted by Ceph on node {{ $labels.node_name }} for more than 10s. Please check pod description for more details.
```

Comment:
Description needs to be more clear that we're talking about pods with `CreateContainerError` error whose pod description has "read-only filesystem" error.

Reply:
ack, updating